### PR TITLE
Module script

### PR DESCRIPTION
**Pull request by tyehle**
- Moves console things to __main__ (this lets you run aft with `python3 -m aft ...`)
- removes dependencies for maximum portability! (click and numpy)
- installs a script with the package (this lets you run with `aft ...`)

### CODE REVIEW

File: aft/test_functions.py [hunk 1]
```diff
@@ -41,7 +41,6 @@ def add_one_only_int_default(a, b=3, c=4):
 class Example(object):
 
     def __init__(self, a, b, c):
-        # type: (int, float, str) -> None
```
Comment (nbertagnolli) [inline aft/test_functions.py:44]: Why did you delete these type annotations?

**Review by nbertagnolli — APPROVED** [review]

File: aft/test_functions.py (hunk 1 above)
Comment (tyehle) [reply aft/test_functions.py:44]: Cause there was a use of the class that didn't conform. Maybe it would be better to change it to `Any`.
